.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -1,51 +1,94 @@
 name: 🐞 Bug Report
-description: File a new bug report
-title: 'bug: <title>'
+description: Create a new bug report.
+title: '🐞 bug: <title>'
 labels: [Bug, Needs Triage]
 body:
+
   - type: markdown
     attributes:
-      value: ':stop_sign: _For support questions, please visit [http://discuss.appium.io](http://discuss.appium.io) instead._'
+      value: ':stop_sign: _Make sure you are reporting an issue. For general questions, please use [http://discuss.appium.io](http://discuss.appium.io)._'
+
   - type: checkboxes
     attributes:
-      label: 'Do I have the most recent component updates?'
```

Comment:
I would prefer to keep this checkbox. Or change its text to something like `I tried to reproduce this issue with the most recent server and driver/plugin versions`

Reply:
Makes sense. I suppose it should be optional, in case the user is already on the latest component versions?

Reply:
maybe. We should also provide information on how to verify if component version is the latest one (e.g. `npm info appium` for the server and `appium X update` CLI for drivers/plugins)

Reply:
I like the command for the server, but the `appium <driver/plugin> update` CLI will also actually update the package, which the user may not want to do right away. But I guess the `npm info <driver/plugin> version` command works here as well?

Reply:
yes, it should do